src/game/client/swarm/c_asw_button_area.h

```diff
@@ -45,7 +45,17 @@ class C_ASW_Button_Area : public C_ASW_Use_Area
 	CNetworkVar(float, m_flHoldTime);
 	
 	virtual const char* GetNoPowerMessage() { return m_NoPowerMessage; }
-	char		m_NoPowerMessage[255];	
+	virtual const char* GetUseIconMessage() { return m_UsePanelMessage; }
```

Comment:
Remove these virtual methods or move them to the cpp file and have them handle the default value automatically.

Reply:
What do you mean by handle the default value automatically?

Reply:
Basically, the functions should look like `GetUseIconText` and not like `GetUseIconMessage`